### PR TITLE
Add card2.0

### PR DESCRIPTION
**Pull request by stevehusmann**
This is all of the changes Nina added, plus I made the add card functionality work in a way that it will be fairly simple to hook it up to the backend. It follows a pretty similar logic as moveCardWithinList

### CODE REVIEW

File: client/src/components/actions/index.js [hunk 1]
```diff
@@ -1,6 +1,7 @@
 import axios from 'axios';
 import uniqid from 'uniqid';
-import { UPDATE_LIST_ORDER, MOVE_CARD_WITHIN_LIST, MOVE_CARD_BETWEEN_LISTS, FETCH_BOARDS, ADD_NEW_LIST, UPDATE_LIST_TITLE } from './types';
+import { useSelector } from 'react-redux';
```
Comment (knp314) [inline client/src/components/actions/index.js:3]: I don't think we are using `useSelector` in this file?

File: client/src/components/actions/index.js [hunk 2]
```diff
@@ -35,11 +36,12 @@ export const fetchBoards = () => dispatch => {
...
 };
 
-export const addNewList = (newListName) => {
+export const addNewList = (newListTitle) => {
+  //The below is just a placeholder until we hook up the backend
   const newListId = uniqid('list-');
   const newList = {
       id: newListId,
-      title: newListName,
+      title: newListTitle,
```
Comment (knp314) [inline client/src/components/actions/index.js:44]: good catch!

File: client/src/components/card.js [hunk 3]
```diff
@@ -12,7 +12,7 @@ const Card = (props) => {
         ref={provided.innerRef}
         isDragging={snapshot.isDragging}
       >
-        {props.card.content}
+        {props.card.title}
```
Comment (knp314) [inline client/src/components/card.js:15]: FYI, this is working fine, but our card model uses `cardTitle` not `title` for this property, so we may have to update the model accordingly

File: routes/main.js [hunk 4]
```diff
@@ -146,13 +146,13 @@ router.get("/boards/:board/lists", (req, res, next) => {
         });   
 });
 
-// POST add a new list to an existing board (working)
+// POST add a new list using its title only to an existing board (working)
```
Comment (knp314) [inline routes/main.js:149]: I'm not sure which is the "correct" routes file, but there is also a routes file under `/server/routes/main.js`, which has a POST card route already, so we should pick one POST card route, and preferably one routes file, too, going forward ...

File: client/src/components/list.js [hunk 5]
```diff
@@ -1,10 +1,11 @@
 import React from 'react';
+import { useState } from 'react';
 import styled from 'styled-components';
 import { Droppable, Draggable } from 'react-beautiful-dnd';
 import { Form , Button} from 'react-bootstrap';
 import useOnClickOutside from 'use-onclickoutside';
-import { useDispatch } from 'react-redux';
-import { updateListTitle } from './actions';
+import { useSelector, useDispatch } from 'react-redux';
```
Comment (knp314) [inline client/src/components/list.js:7]: It doesn't look like we're using `useSelector` here, either?

File: client/src/components/actions/index.js (hunk 1 above)
Comment (stevehusmann) [reply client/src/components/actions/index.js:3]: true true, I was attempting to use it and didn't end up doing so

File: client/src/components/actions/index.js (hunk 2 above)
Comment (stevehusmann) [reply client/src/components/actions/index.js:44]: thx

File: client/src/components/card.js (hunk 3 above)
Comment (stevehusmann) [reply client/src/components/card.js:15]: good catch, changed accordingly

File: client/src/components/list.js (hunk 5 above)
Comment (stevehusmann) [reply client/src/components/list.js:7]: strange, mine doesn't show useSelector

**Pull request closed without merging**